shardingsphere-sql-parser/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/BaseRule.g4

```diff
@@ -328,3 +328,23 @@ ignoredIdentifier_
 ignoredIdentifiers_
     : ignoredIdentifier_ (COMMA_ ignoredIdentifier_)*
     ;
+
+variableExpr
+    : ((scope? variable (TO | EQ_) variableValue) | (scope? timeZone variableValue))
+    ;
+
+variableValue
+    : (identifier | NUMBER_ | (MINUS_ NUMBER_) | STRING_ | DEFAULT | LOCAL)
+    ;
+
+timeZone
+    : TIME ZONE
+    ;
+
+variable
+    : identifier
+    ;
+	
+scope
+    : (SESSION | LOCAL)
+    ;
```

Comment:
Please keep one empty line on the end of file

Reply:
done